### DISCUSSION TITLE
Windows 98 SE Install - Second Hard Drive (VHD) only 1.99gb

### DISCUSSION THREAD

**Discussion by jwood1980** [86Box/86Box#5069] [Q&A] [answered]
Hi everyone, 

I was hoping someone could help me. 

I am using 86Box on my M1 MacBook and have Windows 98 SE installed.

Everything installed and worked fine, however the 20gb HDD (VHD) is only showing as a 1.99gb capacity drive.
I set it up this way, so that I could install software on another drive. But am running out of space when installing a larger software application.

Am I missing something? 

Your help would be appreciated.

I have attached screenshots.››
<img width="418" alt="Screenshot 2024-12-26 at 3 47 03 pm" src="https://github.com/user-attachments/assets/6741fc95-cd2b-48b3-9dcb-8e2a8f8e7769" />
<img width="821" alt="Screenshot 2024-12-26 at 3 48 01 pm" src="https://github.com/user-attachments/assets/ea788de5-c9ad-4a79-8a0a-fda11361a2a5" />

**Comment by lazerode** [comment, marked as answer]
That's because the HDD is formatted under the FAT16 file system, which has a limit of 2047MB. You are supposed to use the FAT32 file system then (on the first screen on FDISK, select Yes to enable FAT32 support).

**Reply by jwood1980** [reply]
Thank you, I completely failed to notice or remember this... Being someone who started on Win 3.1, you would think I would remember the pain involved. hahaha.
